packages/fxa-auth-server/lib/pushbox/index.ts

```diff
@@ -149,13 +157,57 @@ module.exports = function (log, config, statsd, DB = PushboxDB) {
      * @param {String} [index]
      * @returns {Promise}
```

Comment:
These type doc strings aren't needed with TS unless we need additional docs on the param.

Reply:
Plus I think the JSDoc is incorrect since I don't see an `options` argument in the actual signature (5 `@param`s but only 4 arguments).
But not a regression since I don't see `options` object in the previous code below either (only in JSDocs).